lib/DriverTool/autolink_extract_main.cpp

```diff
@@ -164,8 +176,18 @@ extractLinkerFlagsFromObjectFile(const llvm::object::WasmObjectFile *ObjectFile,
       llvm::SmallVector<llvm::StringRef, 4> SplitFlags;
       SegmentData.split(SplitFlags, llvm::StringRef("\0", 1), -1,
                         /*KeepEmpty=*/false);
-      for (const auto &Flag : SplitFlags)
-        LinkerFlags.push_back(Flag.str());
+      for (const auto &Flag : SplitFlags) {
+        // If it is one of the common Swift library flags,
+        // only add if not seen before.
+        if (auto It = CommonAddedSwiftLibraries.find(Flag.str());
+            It != CommonAddedSwiftLibraries.end()) {
```

Comment:
Have we enabled C++17?

Reply:
No we have not.
Wishful thinking on my end. 
Thanks for the catch.